webpack.config.js

```diff
@@ -28,42 +28,69 @@ const commonEntryFiles = {
     background: [path.resolve(__dirname, 'src/background/background-init.ts')],
 };
 
-const commonConfig = {
-    entry: commonEntryFiles,
-    module: {
-        rules: [
-            {
-                test: /\.tsx?$/,
-                use: [
-                    {
-                        loader: 'ts-loader',
-                        options: {
-                            transpileOnly: true,
-                            experimentalWatchApi: true,
+const electronEntryFiles = {
+    main: [path.resolve(__dirname, 'src/electron/main/main.ts')],
+    injected: [path.resolve(__dirname, 'src/injected/stylesheet-init.ts'), path.resolve(__dirname, 'src/injected/client-init.ts')],
+};
+
+const createCommonConfig = entry => {
```

Comment:
Echoing Dan's comment from #686:
`createCommonConfig` is mostly static, except for copying the `entry` parameter to the output as the value for `entry`. Is it more idiomatic to restore `commonConfig` to being a `const`, and change consumers from
```
const electronConfig = {
    ...createCommonConfig(electronEntryFiles),
    name: 'electron',
```
to
```
const electronConfig = {
    ...commonConfig,
    entry: electronEntryFiles,
    name: 'electron',
```

Reply:
Updated.